src/prop/sat_proof_manager.h

```diff
@@ -582,6 +587,22 @@ class SatProofManager
   Node getClauseNode(const Minisat::Clause& clause);
   /** Prints clause, as a sequence of literals, in the "sat-proof" trace. */
   void printClause(const Minisat::Clause& clause);
+
+  /** The user context */
```

Comment:
Is it possible to make this into EnvObj?  Otherwise we will have to clean this later.

Reply:
Will do